tests/foreman/installer/test_infoblox.py

```diff
@@ -15,183 +17,162 @@
+    'foreman-proxy-plugin-dhcp-infoblox-record-type': 'fixedaddress',
+    'foreman-proxy-dns': 'true',
+    'foreman-proxy-dns-provider': 'infoblox',
+    'foreman-proxy-plugin-dns-infoblox-username': settings.infoblox.username,
+    'foreman-proxy-plugin-dns-infoblox-password': settings.infoblox.password,
+    'foreman-proxy-plugin-dns-infoblox-dns-server': settings.infoblox.hostname,
+    'foreman-proxy-plugin-dns-infoblox-dns-view': 'default',
+}
+
+
+@pytest.mark.stream
+@pytest.mark.e2e
+@pytest.mark.parametrize('module_sync_kickstart_content', [8], indirect=True)
+def test_infoblox_end_to_end(
+    request,
+    module_sync_kickstart_content,
+    module_provisioning_capsule,
+    module_target_sat,
+    module_sca_manifest_org,
+    module_location,
+    module_default_org_view,
+    module_lce_library,
+    default_architecture,
+    default_partitiontable,
+):
+    """Verify end to end infoblox plugin integration and host creation with
+       Infoblox as DHCP and DNS provider
+
+    :id: 0aebdf39-84ba-4182-9a17-6eb523f1695f
+
+    :steps:
+        1. Run installer to integrate Satellite and Infoblox.
+        2. Assert if DHCP, DNS provider and server is properly set.
+        3. Create a host with proper domain and subnet.
+        4. Check if A record is created with host ip and hostname on Infoblox
+        5. Delete the host, domain, subnet
+
+    :expectedresults: Satellite and Infoblox are integrated properly with DHCP, DNS
+                      provider and server.Also, A record is created for the host created.
+
+    :BZ: 1813953
+
+    :customerscenario: true
     """
+    enable_infoblox_plugin = InstallerCommand(
+        installer_args=infoblox_plugin_enable,
+        installer_opts=infoblox_plugin_opts,
+    )
+    result = module_target_sat.execute(enable_infoblox_plugin.get_command(), timeout='10m')
+    assert result.status == 0
+    assert 'Success!' in result.stdout
+    assert (
+        module_target_sat.execute(
+            'satellite-installer --help | grep foreman-proxy-dhcp-provider | grep infoblox'
+        ).status
+        == 0
+    )
+    assert (
+        module_target_sat.execute(
+            'satellite-installer --help | grep foreman-proxy-dns-provider | grep infoblox'
+        ).status
+        == 0
+    )
+    assert (
+        module_target_sat.execute(
+            f'satellite-installer --help | grep foreman-proxy-dhcp-server | '
+            f'grep {settings.infoblox.hostname}'
+        ).status
+        == 0
+    )
+    assert (
+        module_target_sat.execute(
+            f'satellite-installer --help | grep foreman-proxy-plugin-dns-infoblox-dns-server | '
+            f'grep {settings.infoblox.hostname}'
+        ).status
+        == 0
+    )
+
+    macaddress = gen_mac(multicast=False)
+    # using the domain name as defined in Infoblox DNS
+    domain = module_target_sat.api.Domain(
+        name='vms.sat.rdu2.redhat.com',
```

Comment:
can we fetch this from settings, instead of hardcoding?

Reply:
Do we have a constant for this?